frontend/src/forms/ParcelDetailForm.test.tsx

```diff
@@ -97,7 +108,7 @@ describe('ParcelDetailForm', () => {
       agencyId: 1,
       address: {
         line1: 'addressval',
-        cityId: 1,
+        cityId: '1',
```

Comment:
Curious change?

Reply:
its to do with moving the logic out of the toApiFormat function and to the individual submits.